var/spack/repos/builtin/packages/opencv/package.py

```diff
@@ -42,8 +42,17 @@ class Opencv(Package):
     homepage = 'http://opencv.org/'
     url = 'https://github.com/Itseez/opencv/archive/3.1.0.tar.gz'
 
-    version('3.2.0', 'a43b65488124ba33dde195fea9041b70')
-    version('3.1.0', '70e1dd07f0aa06606f1bc0e3fa15abd3')
+    version('master', git="https://github.com/opencv/opencv.git", branch="master")
+    version('3.3.0-rc', '16ac824307b130b0fda3c6e4b27dc3d1')
+    version('3.2.0-rc', 'a4ec8f365505dd60b777801e2f3092d5')
```

Comment:
I will remove them.  They were included when I pulled the newer versions of the hash.